README.md

```diff
@@ -25,11 +25,15 @@ The transformations which are already accepted to NL-Augmenter are summarized in
 * [How do I create a filter?](#how-do-i-create-a-filter)
 * [Motivation](docs/doc.md#motivation)
 * [Review Criteria for Accepting Submissions](docs/doc.md#review-criteria-for-submissions)
+* [Informal And Untested Suggestions for Transformations](#informal-and-untested-suggestions-for-transformations)
```

Comment:
I would order the table of contents the same way the text is ordered. So put this right before the Installation bullet.

I would also simplify the section title, to something like:
```suggestion
* [Some Ideas for Transformations](#some-ideas-for-transformations)
```

Reply:
I agree the overall change is helpful. Also agree with Jascha's suggestion above!